example/example_source/example_public_kokkos.cpp

```diff
@@ -15,8 +15,14 @@
 //@HEADER
 
 #include <Kokkos_Core.hpp>
+
+#include <cstdio>
 #include <iostream>
 
+extern "C" void print_fortran_();
```

Comment:
Why do we have a trailing underscore?

Reply:
seems like that is something fortran likes to do ... https://docs.oracle.com/cd/E19957-01/805-4940/z400091044a7/index.html